src/GraphRequest.ts

```diff
@@ -9,6 +9,8 @@
  * @module GraphRequest
  */
 
+import FormData from "form-data";
```

Comment:
I added this line because the [test cases](https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/ed902ecf040d5f2ef644447d6754fb423ac33e0e/spec/development/workload/OneNote.ts#L72)   which used form-data failed because this import wasn't present. I will test this change for a browser sample. But I think the import is needed.

Reply:
Removed the import